features/step_definitions/barclamp_rabbitmq/rabbitmq_steps.rb

```diff
@@ -0,0 +1,74 @@
+Given(/^the barclamp proposal is using rabbitmq as messaging backend$/) do
+    json_response = JSON.parse(admin_node.exec!("crowbar rabbitmq show default").output)
+    expect(json_response["attributes"]["rabbitmq"]).not_to be_empty
+end
+
+When(/^the node with rabbitmq-server role has been detected successfully$/) do
+    @node = nodes.find(barclamp: "rabbitmq", element: "rabbitmq-server").first
```

Comment:
Why not to take all nodes here and do the tests for all of them? It will make the tests more comprehensive.